### DISCUSSION TITLE
Upload to F-Droid

### DISCUSSION THREAD

**Discussion by ItsLiyua** [ReVanced/revanced-discussions#426] [Question]
Maybe upload an all-in-one build with all patches (and the manager once it's done) to F-Droid for easier install and updating.

**Comment by SinTan1729** [comment]
From my understanding, it's a no-no to upload apks. Otherwise, this project may end up with the same fate as the original Vanced. Having said that, I'm pretty sure there'll be tons of unofficial sources for that, use them with precaution though. I believe once the Manager app is finished, you won't need any prebuilt apks anymore.

**Reply by alexandreteles** [reply]
A good option might be to create a template repository that builds the APKs for you using GitHub Actions. Just have to make sure to not run that in a public repo. That's what I'm doing currently so I always have the latest build available to me.

**Reply by anoraktrend** [reply]
Revanced manager is unmaintained

**Reply by SinTan1729** [reply]
@anoraktrend As others have mentioned, there are various repos with all the tools one needs to build ReVanced automatically. For example, my repo [ReVancedBuilder](https://github.com/SinTan1729/ReVancedBuilder) lets you build ReVanced automatically (checking regularly for updates) and optionally sends a notification through telegram, and do additional things via a post run script.
